src/jit/emit.h

```diff
@@ -2122,6 +2123,8 @@ class emitter
         VarSetOps::AssignNoCopy(emitComp, emitInitGCrefVars, VarSetOps::MakeEmpty(emitComp));
         VarSetOps::AssignNoCopy(emitComp, emitThisGCrefVars, VarSetOps::MakeEmpty(emitComp));
     }
+
+    regMaskTP GetSavedSet(CORINFO_METHOD_HANDLE methHnd);
```

Comment:
IMO, this declaration should be up higher, maybe in the `The following logic keeps track of live GC ref values` section

Reply:
Fixed